common/common.h

```diff
@@ -482,6 +482,8 @@ struct common_params {
 
     // "advanced" endpoints are disabled by default for better security
     bool webui            = true;
+    std::string webui_config_file;
+    std::string webui_config_json;
     bool endpoint_slots   = true;
     bool endpoint_props   = false; // only control POST requests, not GET
     bool endpoint_metrics = false;
```

Comment:
You can arrange these a bit better like so:

```suggestion
    // webui configs
    bool webui = true;
    
    std::string webui_config_file;
    std::string webui_config_json;
    
    // "advanced" endpoints are disabled by default for better security
    bool endpoint_slots   = true;
    bool endpoint_props   = false; // only control POST requests, not GET
    bool endpoint_metrics = false;
```